first-analysis-steps/physics-at-lhcb.md

```diff
@@ -0,0 +1,147 @@
+# Physics at LHCb
+
+{% objectives "Learning Objectives" %}
+
+* Understand how detector signals are turned into objects in software
+* Learn how decay chains are built.
+
+Note that this is a very high level introduction to a high-energy physics
+experiment, and misses many important details.
+
+{% endobjectives %} 
+
+The software that these lessons will cover was designed to make the types of
+physics analyses most commonly performed at LHCb as easy as possible. When
+using that software, it is then important to understand how those analyses are
+done in order to understand why the software works the way it does.
+
+At the highest level, a physics analysis tries to measure one or more
+properties of objects produced through some process. At LHCb, this is typically
+some production or decay properties of heavy flavour hadrons. Because these
+objects have short lifetimes they decay before interacting with the detector,
+and so we must infer their properties through their decay products. Particles
+such as charged kaons and pions have lifetimes large enough that a large
+fraction can traverse the full detector, and so we typically consider these
+particles as ‘stable’. Unstable objects, with much shorter lifetimes, are
+formed as combinations of these ‘stable’ particles:
+
+1. Charged pions $$\pi^{\pm}$$
+2. Charged kaons $$K^{\pm}$$
+3. Protons $$p/\bar{p}$$
+4. Electrons $$e^{\pm}$$
+5. Muons $$\mu^{\pm}$$
+6. Photons $$\gamma$$
+
```

Comment:
A nit-picky person might include $$K_{L}$$ here, although we don't actually ever build these. I think they would look like a MIP in the calorimeter and then have muon hits.

Reply:
By that logic, one should include deuterons too. Probably even neutrons. But it's better to keep things simple.

Reply:
I'd like to list everything that's in the output of the standard reconstruction. Deuterons have been present for a while so I'll add those. I'll add a footnote about neutrons and KLs (we should be nit-picky people :smile: ).